browser/src/VariantPage/HGDPPopulationsTable.tsx

```diff
@@ -38,18 +38,23 @@ const HGDP_POPULATION_GROUPS = {
     'makrani',
     'pathan',
     'sindhi',
+    'uygur',
   ],
+  'Oceanian': ['bougainville', 'papuanhighlands', 'papuansepik'],
 }
 
 const HGDP_POPULATION_NAMES = {
```

Comment:
I'm not positive that I'm the best one to look at this part. Can we have Mary take a quick look over it too?

Reply:
I will ask her, the changes are really in the keys, the values can be more understandable, because they need to use our HT, the correct updated keys are important.

Reply:
She seems to be OOO today, do we wait?

Reply:
No, maybe @ch-kr can weigh in?

Reply:
I made a couple suggestions, I think it all looks good otherwise, I just want to make sure the display names look OK so if @ch-kr confirms that it's good to go

Reply:
these look good to me; I also quickly checked pages 10 and 11 of the supplement from https://www.science.org/doi/10.1126/science.aay5012?url_ver=Z39.88-2003&rfr_id=ori%3Arid%3Acrossref.org&rfr_dat=cr_pub++0pubmed&

if we want someone from Alicia's team to review, perhaps we could ask Zan?

Reply:
Mary replied, she can only get onto it next Monday, latest EOD Monday, but I think it's good enough.